applications/crossbar/src/modules/cb_conferences.erl

```diff
@@ -64,14 +64,25 @@ init() ->
 -spec allowed_methods() -> http_methods().
 -spec allowed_methods(path_token()) -> http_methods().
 -spec allowed_methods(path_token(), path_token()) -> http_methods().
-allowed_methods() ->
-    [?HTTP_GET, ?HTTP_PUT].
-allowed_methods(_) ->
-    [?HTTP_GET, ?HTTP_POST, ?HTTP_PATCH, ?HTTP_DELETE].
-allowed_methods(_, <<"details">>) ->
-    [?HTTP_GET];
-allowed_methods(_, <<"mute">>) ->
-    [?HTTP_POST].
+-spec allowed_methods(path_token(), path_token(), path_token()) -> http_methods().
+
+allowed_methods() -> [?HTTP_GET, ?HTTP_PUT].
+
+allowed_methods(_) -> [?HTTP_GET, ?HTTP_POST, ?HTTP_PATCH, ?HTTP_DELETE].
+
+allowed_methods(_, <<"details">>) -> [?HTTP_GET];
+
+allowed_methods(_, <<"lock">>) -> [?HTTP_POST];
```

Comment:
Also, in `allowed_methods`, please don't use `_` for variables. `cb_api_endpoints` uses those names to generate the ref docs. `_` will become `{ID}` which isn't too instructive. `_ConferenceId` would become `{CONFERENCE_ID}` which is much nicer. Also, if you match explicit tokens, please do so explicitly.